GVFS/GVFS.Common/NamedPipes/NamedPipeMessages.cs

```diff
@@ -158,9 +158,25 @@ public Request(Message message)
                     this.UpdatedSkipWorktreeBits = message.Body[1] == '1';
                 }
 
+                public Request(bool updatedWorkingDirectory, bool updatedSkipWorktreeBits)

```

Comment:
Just for others to avoid questions: the hook uses the `public Request(Message message)` overload above, so this new constructor does not require reaction work there.